streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -458,9 +521,11 @@ public void foreach(final ForeachAction<? super K, ? super V> action, final Name
     }
 
     @Override
-    public KStream<K, V> through(final String topic, final Produced<K, V> produced) {
+    public KStream<K, V> through(final String topic,
+                                 final Produced<K, V> produced) {
         Objects.requireNonNull(topic, "topic can't be null");
-        Objects.requireNonNull(produced, "Produced can't be null");
+        Objects.requireNonNull(produced, "produced can't be null");
```

Comment:
fixed typo (was always use lower case letter as we refer to the parameter name -- there was multiple typos on those error messages -- won't highlight all)